static/app/components/sidebar/sidebarDropdown/switchOrganization.spec.tsx

```diff
@@ -35,42 +47,10 @@ describe('SwitchOrganization', function () {
     jest.useRealTimers();
   });
 
-  it('uses organizationUrl when customer domain is enabled', async function () {
```

Comment:
I removed this test because the `multi-region` flag cannot be turned off at the organization level like `customer-domain` could be.